RedHat-UX/red-hat-design-system
# docs(stat): spacer table

## What I did

1.  Updated the stat docs to use the new spacer table code
2. Removed previous images for spacers from stat.


## Testing Instructions

1.  Please check the deploy preview and verify that the spacer table looks correct for `rh-stat`.
